resources/server_configuration.rb

```diff
@@ -40,6 +40,7 @@
 property :mysqld_skip_external_locking,   [true, false],     default: true
 property :mysqld_skip_log_bin,            [true, false],     default: false
 property :mysqld_skip_name_resolve,       [true, false],     default: false
+property :mysqld_port,                    Integer,           default: 3306
```

Comment:
could we just pot, port, address, etc? Rather than preappending everything with mysqld?